### PR TITLE
Refactor/tier 2 xdr viewer tsx

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: src/views/__tests__/XdrViewer.test.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jsdom
+ */
+
```
Comment (piyalbasu) [inline src/views/__tests__/XdrViewer.test.tsx:4]: I had been holding off on merging because I wanted to add more tests, but I can do that in a later PR

**Review by quietbits — APPROVED** [review]

**Comment by acharb** [comment]
@piyalbasu I was playing around with it and I'm getting a `Some source accounts don't exist. Are you on the right network?` message when entering XDR into the the view XDR input. eg
```
AAAAAgAAAABY+dh/7qRA3e3ji+B6xB1aHgk44+Ptp6vznYjtEmG3MAAAB9IAAI8oAAAAQgAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAABY+dh/7qRA3e3ji+B6xB1aHgk44+Ptp6vznYjtEmG3MAAAAAAAAAAAAJiWgAAAAAAAAAAA
```
looks like the request gets sent like: `http://localhost:3000/undefined/accounts/GBMPTWD752SEBXPN4OF6A6WEDVNB4CJY4PR63J5L6OOYR3ISMG3TA6JZ ` 

is that caused by a change from this PR or something else? (I tried and failed finding where this request is called from)

**Comment by piyalbasu** [comment]
> ```
> AAAAAgAAAABY+dh/7qRA3e3ji+B6xB1aHgk44+Ptp6vznYjtEmG3MAAAB9IAAI8oAAAAQgAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAABY+dh/7qRA3e3ji+B6xB1aHgk44+Ptp6vznYjtEmG3MAAAAAAAAAAAAJiWgAAAAAAAAAAA
> ```

Argh good catch!! This was a bug that I just fixed. 

+1 for completing tests before opening a PR 🤦

**Review by acharb — APPROVED** [review]
lgtm!

**Pull request merged**